src/main/java/br/com/alura/comex/CategoriasProcessadas.java

```diff
@@ -1,6 +1,49 @@
 package br.com.alura.comex;
 
-import java.util.HashSet;
+import java.util.ArrayList;
+import java.util.List;
 
-public class CategoriasProcessadas extends HashSet<String> {
+public class CategoriasProcessadas {
+
+  ArrayList<String> categorias;
```

Comment:
Atributos privados. Prefira interfaces a implementações, quando possível.

```suggestion
  private List<String> categorias;
```